<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogDirectoryPicker</class>
 <widget class="QWidget" name="LogDirectoryPicker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>209</width>
    <height>32</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLineEdit" name="pathTextBox"/>
   </item>
   <item>
    <widget class="QLabel" name="pathNotFound">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string>This path could not be found. It will not be used.</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../../../../media/resources.qrc">:/icons/exclamation.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnLogBrowse">
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbLoggingEnabled">
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Enabled</string>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../media/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnLogBrowse</sender>
   <signal>clicked()</signal>
   <receiver>LogDirectoryPicker</receiver>
   <slot>browse()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>170</x>
     <y>15</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbLoggingEnabled</sender>
   <signal>toggled(bool)</signal>
   <receiver>LogDirectoryPicker</receiver>
   <slot>cbLoggingEnabledChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>170</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pathTextBox</sender>
   <signal>editingFinished()</signal>
   <receiver>LogDirectoryPicker</receiver>
   <slot>textEdited()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>32</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>31</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>browse()</slot>
  <slot>cbLoggingEnabledChecked(bool)</slot>
  <slot>textEdited()</slot>
 </slots>
</ui>
